src/main/java/seedu/address/model/person/Sex.java

```diff
@@ -10,8 +12,8 @@
 public class Sex {
 
     public static final Sex DEFAULT = new Sex("F");
-    public static final String MALE = "M";
-    public static final String FEMALE = "F";
+    public static final String MALE = "m";
+    public static final String FEMALE = "f";
```

Comment:
@tengcharmaine doing this will cause the 'isFemaleString()' / 'isMaleString()' check to fail. You probably either need to make sex upper case after checking `isValidSex`, or modify how `isFemaleString` and `isMaleString` works to check both upper and lower case sexs.